packages/frontend/editor-ui/src/composables/useCanvasOperations.ts

```diff
@@ -1974,14 +1973,10 @@ export function useCanvasOperations({ router }: { router: ReturnType<typeof useR
 		return data;
 	}
 
-	async function toggleChatOpen(source: 'node' | 'main') {
+	async function toggleChatOpen(source: 'node' | 'main', isOpen?: boolean) {
```

Comment:
Nit: It seems weird to pass `isOpen` flag to a function called `toggle`. Can we go without it and deduce the state inside the function?

Reply:
> deduce the state inside the function

Yes this is what happens when isOpen is omited. I think it's more or less a common practice to allow optionally providing "forced" state as an argument in toggle function. An example in Element Plus: `toggleRowSelection` in https://element-plus.org/en-US/component/table.html#table-exposes